3.0/browsers/Dockerfile

```diff
@@ -59,5 +31,33 @@ ENV DISPLAY=":99"
 RUN	printf '#!/bin/sh\nXvfb :99 -screen 0 1280x1024x24 &\nexec "$@"\n' | sudo tee /docker-entrypoint.sh && \
 	sudo chmod +x /docker-entrypoint.sh
 
+# Install a single version of Firefox. This isn't intended to be a regularly
+# updated thing. Instead, if this version of Firefox isn't want the end user
```

Comment:
Small typo 🙂 

```suggestion
# updated thing. Instead, if this version of Firefox isn't what the end user
```

Reply:
thanks @mattbrictson  - would you mind putting in a PR for https://github.com/CircleCI-Public/cimg-shared/blob/main/variants/browsers.Dockerfile.template and we can have you added in as a contributor?